src/core_plugins/kibana/public/discover/components/field_chooser/lib/detail_views/string.html

```diff
@@ -44,6 +44,7 @@ <h5 ng-show="!field.details.error">Quick Count <kbn-info info="Top 5 values base
 
       <a
         ng-href="{{vizLocation(field)}}"
+        ng-if="field.aggregatable"
```

Comment:
I think this would be more ergonomic if it called a method, like this:

```
ng-if="canFieldBeVisualized(field)"
```

which then called a service:

```
import isFieldVisualizable from 'ui/visualize/is_field_visualizable'; // location tbd
scope.canFieldBeVisualized = canFieldBeVisualized;
```

which codified this rule (and future related rules):

```
// is_field_visualizable.js
export default isFieldVisualizable(field) {
  // Some explanation here about why non-aggregatable fields can't be visualized.
  if (!field.aggregatable) {
    return false;
  }

  return true;
}
```

It seems like a lot of code for such a small feature, but I think it's important we codify our business logic this way. Otherwise the "what" of the code isn't justified by a "why". Other benefits include increased readability, testability (we can test the service), and scalability (we can add additional rules to the service).